object/ldap_conn.go

```diff
@@ -376,6 +404,11 @@ func SyncLdapUsers(owner string, syncUsers []LdapUser, ldapId string) (existUser
 	}
 	tag := strings.Join(ou, ".")
 
+	existingEmails, err := GetExistingEmails(owner, emails)
```

Comment:
This logic should only work when turned  application.EnableLinkWithEmail

Reply:
What about background sync?